tools/tsp-client/src/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function formatAdditionalDirectories(additionalDirectories?: string): string {
```

Comment:
should this be an string[] instead of string?

Reply:
It should be string[], just pushed the updated type for this parameter. Thanks for pointing that out :)